torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -3515,11 +3515,10 @@ at::Tensor XLANativeFunctions::embedding_symint(const at::Tensor& weight,
     return at::native::embedding_symint(weight, indices, padding_idx,
                                         scale_grad_by_freq, sparse);
   }
-  // TODO: for now route to native, which dispatches supported XLA operations.
-  // We need to make use of the TPU embedding core here eventually.
```

Comment:
This comment (seems like it existed for a while) `We need to make use of the TPU embedding core here eventually.` makes me a bit cautious about the `Embedding` op. I assume this comment means to lower this op in a performant way, we somehow want to make use of the TPU embedding core? Just leaving this comment as a future reference. Also let's bring this comment back in the code so we are aware. Again, not a blocker, we can include as a follow-up in the next PR.

Reply:
Sounds good. Will add it back in a follow up PR